.github/workflows/pull-request.yml

```
@@ -312,3 +312,15 @@ jobs:
+  Import-Sample-Data:
+    name: Import Sample Data
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v4.2.2
+      - name: Create .env file for talawa api testing environment
+        run: cp ./envFiles/.env.ci ./.env
+      - name: Build talawa api non production environment docker image
+        run: docker compose build
+      - name: Import sample data
+        run: docker exec talawa-api-1 /bin/bash -c "pnpm run import:sample-data"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)